editor/block-switcher/index.js

```diff
@@ -72,38 +47,21 @@ class BlockSwitcher extends Component {
 		}
 
 		return (
-			<div className="editor-block-switcher">
-				<IconButton
-					className="editor-block-switcher__toggle"
-					icon={ blockType.icon }
-					onClick={ this.toggleMenu }
-					aria-haspopup="true"
-					aria-expanded={ this.state.open }
-					label={ __( 'Change block type' ) }
-				>
-					<Dashicon icon="arrow-down" />
-				</IconButton>
-				{ this.state.open &&
-					<div
-						className="editor-block-switcher__menu"
-						role="menu"
-						tabIndex="0"
-						aria-label={ __( 'Block types' ) }
-					>
-						{ allowedBlocks.map( ( { name, title, icon } ) => (
-							<IconButton
-								key={ name }
-								onClick={ this.switchBlockType( name ) }
-								className="editor-block-switcher__menu-item"
-								icon={ icon }
-								role="menuitem"
-							>
-								{ title }
-							</IconButton>
-						) ) }
-					</div>
-				}
-			</div>
+			<Toolbar>
+				<li>
```

Comment:
It would make using the Toolbar with children slightly easier (and I have considered it), however it would also remove some control from the caller (ie to add classes to some li elements).